src/lv_objx/lv_label.c

```diff
@@ -558,6 +603,31 @@ uint16_t lv_label_get_letter_on(const lv_obj_t * label, lv_point_t * pos)
     return lv_encoded_get_char_id(txt, i);
 }
 
+int lv_label_get_selection_start( const lv_obj_t * label ) {
```

Comment:
Add comments to the function, please.

`int` should be replaced by `int32_t` because `int` is not used in the library directly. (I know that it was in the original implementation too)

Reply:
fixed